plugins/cocktail-lookup.js

```diff
@@ -0,0 +1,106 @@
+const Doorman = require('../../doorman');
```

Comment:
That's exactly what I've done with my bot, [Kirbi](https://github.com/Richardson-Media-House/Kirbi)